libs/blocks/marquee/marquee.css

```diff
@@ -718,7 +718,7 @@
   color: inherit;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 900px) {
```

Comment:
Media queries should go in a natural order. aka 900 should come before 1200. 

@Axelcureno  - Where is the example of this in a marquee block. The `.marquee.mnemonic-list ` selector indicates this is a variant of the marquee? We shouldn't separate concerns and not have nested block styles where ever possible. Big picture, Is this even needed. What happens if your in a marquee.quiet w/ out a split content?

Reply:
I removed this piece of code since it's not required anymore (confirmed this with Consonant and product team).
The example of the Marquee using this block can be found here: https://main--cc--adobecom.hlx.page/drafts/axel/catalog-with-mnemonic